### PR TITLE
Applied resources script to patched-vscode

### PR DESCRIPTION
**Pull request by aws-spenceng**
*Issue #, if available:*

*Description of changes:* 
In the last PR, I made a fix to the `.copy-resources.sh` script that worked in moving the files in the `vscode` folder but I did not duplicate the folder to `patched-vscode`. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-dark.svg
Comment (edwardps) [inline patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-dark.svg:1]: Can you add screenshot when this icon is used? The added seems to be empty.

File: patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-hcLight.svg
Comment (edwardps) [inline patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-hcLight.svg:1]: same question on how it looks in the real UI

File: patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-dark.svg
Comment (aws-spenceng) [reply patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-dark.svg:1]: @edwardps you can preview the icon difference by selecting "Display the Rich diff" button 

Screenshot attached: 
image

File: patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-hcLight.svg
Comment (aws-spenceng) [reply patched-vscode/src/vs/workbench/browser/parts/editor/media/letterpress-hcLight.svg:1]: Same as above, attached screenshot 
image

**Pull request closed without merging**